src/HttpHdrSc.cc

```diff
@@ -134,7 +134,7 @@ HttpHdrSc::parse(const String * str)
 
         if (sct->isSet(type)) {
             if (type != SC_OTHER)
-                debugs(90, 2, "hdr sc: ignoring duplicate control-directive: near '" << item << "' in '" << str << "'");
+                debugs(90, 2, "hdr sc: ignoring duplicate control-directive: near '" << item << "' in '" << *str << "'");
```

Comment:
Same reasoning as in the other similar change request:

```suggestion
                debugs(90, 2, "ignoring duplicate control-directive near '" << item << "' in '" << *str << "'");
```